src/pages/Main/pages/SpendingDetail/SpendingDetail.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { BiArrowBack, BiPlus } from 'react-icons/bi';
+import { GrFormDown } from 'react-icons/gr';
+import './SpendingDetail.scss';
+
+const SpendingDetail = () => {
+  const navigate = useNavigate();
+
+  return (
+    <div className="spendingDetail">
+      <header className="detailHeader">
+        <BiArrowBack
+          className="headerIcon"
+          onClick={() => {
+            navigate(-1);
+          }}
+        />
+        <div className="headerDate">
+          2023년 9월 <GrFormDown />
+        </div>
+        <BiPlus className="headerIcon" />
+      </header>
+
+      <main className="detailMainContents">
+        <section className="totalAmountSection">
+          <p className="sectionTitle">총 지출</p>
+          <h3 className="totalAmount">889,920 원</h3>
+        </section>
+        <section className="breakdownSection">
+          <p className="breakdownSummary">
+            총 <span className="summaryNumber">3</span> 건
+          </p>
+          {/* 아래 카드들은 컴포넌트로 분리 후 날짜별.map 안에 카드들.map 돌릴 수 있을 듯 */}
```

Comment:
`TODO`, `TOFIX` 등을 앞에 붙여주시면 추후에 쉽게 찾아서 관리할 수 있습니다.
```suggestion
          {/* TODO: 아래 카드들은 컴포넌트로 분리 후 날짜별.map 안에 카드들.map 돌릴 수 있을 듯 */}
```